boot/freeldr/freeldr/include/debug.h

```diff
@@ -125,10 +125,12 @@ void    MEMORY_WRITE_BREAKPOINT4(unsigned long addr);
 
 #endif // DBG
 
+DECLSPEC_NORETURN
 void
 NTAPI
 FrLdrBugCheck(ULONG BugCode);
 
+DECLSPEC_NORETURN
```

Comment:
Please also update the function there:
- https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/arch/i386/i386bug.c;hb=848ad61bba06668e0c58cfa76ab5b63eea76a87d#l197
- https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/arch/arm/debug.c;hb=848ad61bba06668e0c58cfa76ab5b63eea76a87d#l27

Reply:
Done.